### DISCUSSION TITLE
JavaMelody and Jira Server 8.20.x compatibility

### DISCUSSION THREAD

**Discussion by bertorga** [javamelody/javamelody#1076] [General]
Hello,

At https://marketplace.atlassian.com/apps/20909/javamelody-monitoring-plugin/version-history, it seems that the last version for jira Server was 1.87.0 because the later versions are for Jira Data Center. 

Do you know if they are also compatible for Jira Server? 

Thank you very much

**Comment by evernat** [comment]
All versions of  the javamelody plugin for JIRA/Confluence/Bamboo/Bitbucket are compatible with JIRA/Confluence/Bamboo/Bitbucket server and with JIRA/Confluence/Bitbucket data center, whatever is displayed in Atlassian marketplace.

Personnallly, I have stopped trying to workaround or to understand the issues in the Atlassian marketplace.
